test/conformance/testing/include/uur/utils.h

```diff
@@ -170,7 +170,19 @@ inline std::string GetDeviceName(ur_device_handle_t device) {
 }
 
 inline std::string GetPlatformAndDeviceName(ur_device_handle_t device) {
-    return GetPlatformName(GetPlatform()) + "__" + GetDeviceName(device);
+    constexpr int MaxAddressLength = 256;
```

Comment:
hmm shouldn't you use `snake_case`... for var names?

Reply:
You're right